### DISCUSSION TITLE
Is it possible to link from a markdown document to a generated chapter in latex/pdf

### DISCUSSION THREAD

**Discussion by HoT-1** [doxygen/doxygen#11129] [Q&A]
I am generating Latex/PDF output and I added a markdown document with an introduction which briefly explains the structure of the document.
But I cannot figure out how to insert insert links into the markdown which point to generated chapters e.g. Design Unit Hierarchy.

I tried \ref and markdown link syntax but the generated chapters don't seem to have labels.
All I managed to do was linking to hierarchy.html file in html output, but that does help with linking within the PDF.

Any Ideas?

**Comment by albert-github** [comment]
To be able to help you:
- Can you please attach a, small, self contained example (source+configuration file in a, compressed, tar or zip file!) that allows us to reproduce the problem? Please don't add external links as they might not be persistent (also references to GitHub repositories are considered non persistent).
- Please also specify the **full** doxygen version used (`doxygen -v`).
